package/Dockerfile

```diff
@@ -10,8 +10,10 @@ RUN zypper --non-interactive update \
 RUN git clone -b v0.6.8 --depth 1 https://github.com/aquasecurity/kube-bench.git
 
 FROM registry.suse.com/bci/bci-base:15.4
+ARG kubectl_version=1.24.10
 ARG kube_bench_tag=0.6.8
 ARG sonobuoy_version=0.56.7
+ARG ARCH=amd64
```

Comment:
The `ARCH` arg is taken from the build environment described by @orangedeng and should work since we are using it [here](https://github.com/rancher/security-scan/blob/master/.drone.yml#L70).

Is it possible to run a test build, to verify?

Reply:
Alright! Let me build an image with your changes and test it on a cluster with ARM nodes.

Reply:
It does not support cross-compile / cross-build; make sure to build on arm64 nodes.

Reply:
@rayandas how did your test end up?

Reply:
It's working fine on ARM cluster.  Cloud you please fix the conflicts?